ocaml/xapi/xapi_vm_clone.ml

```diff
@@ -231,6 +231,8 @@ let quiesced = "quiesced"
 
 let snapshot_info ~power_state ~is_a_snapshot =
   let power_state_info =
+    (* Uses a shadowed function from record_util because xapi_vm_clone.mli
+       interface includes non-datamodel enum variants *)
```

Comment:
I can't find where the non-datamodel enum variants are used. Are they really useful?